src/utils/types.ts

```diff
@@ -0,0 +1,35 @@
+export interface FirebasePlayerInfo {
```

Comment:
I've thought about this a bit and maybe we should try something like this:
```ts
interface FirebasePlayerInfo;
interface FirebasePlayerInfoWithTurn;
interface FirebasePlayerInfoWithCards;
```

Reply:
Allright, I have used the suggested names, thanks.